pyiron_base/jobs/job/generic.py

```diff
@@ -632,10 +633,15 @@ def write_input(self):
         Call routines that generate the code specific input files
         Returns:
         """
-        write_input_files_from_input_dict(
-            input_dict=self.get_input_parameter_dict(),
-            working_directory=self.working_directory,
-        )
+        if not self._input_is_written:
```

Comment:
How about checking if the working directory is empty and only write the input files when it is:
```suggestion
        if len(os.listdir(self.working_directory)) != 0:
```